### ISSUE TITLE
[1.10.2] - Phantom GP usage - DW20 Pack

### ISSUE THREAD

**Issue by justkunas** [rwtema/extrautilities_old_issues#1115]
With one chunk loading ward in the nether the amount of GP usage just keeps increasing despite not adding anything onto the network.

Attached is an image of the power monitor (with calculations) to show what should be happening along with what actually is happening.

If I remove the one in the nether the GP return to normal.

![extra utils bug](https://cloud.githubusercontent.com/assets/7889229/20569214/350d14be-b197-11e6-92a0-c17fcd753559.png)

I am running the DW20 1.10.2 pack with extreme reactors installed.

Note: as soon as I enter the nether it resets to normal and stays that way untill I re-enter the overworld

**Comment by ghost** [comment]
I can't recreate the problem. Can you replicate the issue on a new world?

I also can't recreate the pipe or water mill setup to get those numbers. Could you post a picture of your water mill set up and list the upgrades in those transfer nodes?

Then try breaking various mills and users one at a time and see if the problem clears and if it returns when you replace them.

**Comment by platinawolf** [comment]
I've gotten the same issue... Only a bit worse... Breaking all generators and consumers I'm at 29396/0. This includes dropping the angel ring on the floor. I used windmills, watermills, 3 chunkloading wards, an angel ring (dragon) as well as a resonator with 20 speed upgrades...

Same pack, with ProjectE added to the mix.
Renaming/removing the XU2SaveData.dat file fixed the issue, tossing in the corrupted savedata, maybe someone can read it and figure out exactly what is wonky with it?

[XU2SaveData.dat.zip](https://github.com/rwtema/extrautilities/files/666032/XU2SaveData.dat.zip)

**Comment by ghost** [comment]
Can you give me a list of all the generators/consumers you were running and their power use/gen? I'll play around in creative and see if I can find the culprit.

**Comment by platinawolf** [comment]
Erh,,, Not exactly,,, 36+ windmills,,, 20 watermills? Configuration of watermills was as following:

W----W
-M-M-
--W--
-M-M-
W---W
-M-M-
--W--
-M-M-
W---W
-M-M-
--W--
W is sourceblock and M is watermill. One of them was on grass and the other on stone brick.
2 arrays of these in different chunks. Each water mill gave 16 energy. Windmill goes up and down a ton.

Consumers:
3 chunk wards (One in nether)
Angel ring (dragon wings)
Resonator with 20 speed upgrades.

**Comment by ghost** [comment]
I can work with that. I'll see if I can isolate the problem. Thanks.

**Comment by Phonavis** [comment]
This issue is also in hermitcraft with Transfer Nodes and Mining upgrades, I went to the nether and was flying around with an angel ring and my GP usage was slowly climbing with nothing but a Transfer node with a mining upgrade currently active. I was chunk loading with ftb Utils rather than a ward however...

**Comment by ghost** [comment]
Yeah, I didn't get a chance to look at it before Tema released the 1.1.3 update. But, that can't run with TCon, so it needs a patch already.

**Comment by stormjacque44** [comment]
I am having a similar issue. It's gotten to the point where it's disrupting resource gathering (I use Mechanical Users with my farm) I tore down everything that uses GP to try and figure out what's going on, leaving just my Water Mills (generating 512GP) Even with NO blocks using GP, I'm still registering a 360 GP usage. I haven't even made an angel ring yet.

EDIT: I'm playing on Sky Factory 3 v3.0.6
